### PR TITLE
Update content model article for v8

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to: https://github.com/umbraco/UmbracoDocs/issues/3263

### CODE REVIEW

**Comment by busrasengul** [comment]
Looks so good @erikjanwestendorp 
Let us review this first then merge 🎉

File: Reference/Management/Models/Content.md [hunk 1]
```diff
@@ -65,7 +64,7 @@ return content.CreatorId;
 
 ### .ContentType
 
-Returns a `ContentType` object representing the DocumentType used by the given `Content`.
+Returns a `ISimlpeContentType` object representing the DocumentType used by the given `Content`.
```
Comment (busrasengul) [inline Reference/Management/Models/Content.md:67]: Would you mind double checking the return object itself @erikjanwestendorp 
I assume there's a typo here :)

File: Reference/Management/Models/Content.md [hunk 2]
```diff
@@ -212,14 +175,14 @@ var content = contentService.GetById(1234);
...
-### .ReleaseDate
+### .PublishDate
 
 If set, returns `DateTime` indicating when the `Content` should be published and made available on the website and cache.
 
 ```csharp
 // Given a `ContentService` object get Content by its Id and set the release date to 4 days from now
 var content = contentService.GetById(1234);
-content.ReleaseDate = DateTime.Now.AddDays(4);
+return content.ReleaseDate
```
Comment (busrasengul) [inline Reference/Management/Models/Content.md:185]: ReleaseDate shouldn't be here I guess?

Comment (erikjanwestendorp) [reply Reference/Management/Models/Content.md]: @busrasengul You are absolutely right, just updated.

File: Reference/Management/Models/Content.md (hunk 1 above)
Comment (erikjanwestendorp) [reply Reference/Management/Models/Content.md:67]: @busrasengul As far as I can see it returns ISimlpeContentType. 
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Core/Models/ContentBase.cs#L83

Comment (busrasengul) [reply Reference/Management/Models/Content.md:67]: ISimpleContentType then but your PR says ISimlpeContentType. So there's a typo 😊 (l and p swapped places)

Comment (erikjanwestendorp) [reply Reference/Management/Models/Content.md]: Hahah oke, my bad 🙈. Just fixed it!

**Pull request merged**